__tests__/app.test.js

```diff
@@ -18,4 +20,32 @@ describe("GET /api", () => {
         expect(endpoints).toEqual(endpointsJson);
       });
   });
+
+
+});
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(Array.isArray(topics)).toBe(true);
+        topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String)
+          });
+        }); 
+      });
+  });
+
+  test("404: Responds with a message of 'Bad Request' when the URL is not found", () => {
```

Comment:
This is a test that has to happen, however it is not anything to do with topics and should have its own describe block for a missing endpoint.

Reply:
I've moved to a new describe block.